vignettes/datatable-benchmarking.Rmd

```diff
@@ -131,6 +198,25 @@ setindex(DT, a)
 # }
 ```
 
-# inside a loop prefer `setDT` instead of `data.table()`
+### inside a loop prefer `setDT()` instead of `data.table()`
+
+As of now `data.table()` has an overhead, thus inside loops it is preferred to use `as.data.table()`, even better `setDT()`, or ideally avoid class coercion as described in _avoid class coercion_ above.
```

Comment:
```suggestion
### inside a loop prefer `setDT()` or `as.data.table()` instead of `data.table()`

As of now `data.table()` has an overhead, thus inside loops it is preferred to use `as.data.table()`, even better `setDT()`, or ideally avoid class coercion as described in _avoid class coercion_ above.
```

Reply:
No bug, I don't think we should be focusing on optimizing it, as.data.table is meant to be fast.